tests/brevitas/core/test_standalone_view_wrapper.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+import warnings
+import torch
+import torch.nn as nn
+
+from brevitas.core.stats.view_wrapper import _ViewParameterWrapper, _ViewCatParameterWrapper
+from brevitas.core.stats.stats_op import AbsMax
+
+
+def test_scaling_state_dict():
```

Comment:
Could you please create two tests:
- One called test_scaling_state_dict_viewparameterwrapper with the class TestModuleStd and the respective assert
- One called test_scaling_state_dict_viewcatparameterwrapper with the class TestModuleCat and the respective assert